kyo-core/shared/src/test/scala/kyoTest/streamsTest.scala

```diff
@@ -321,7 +321,7 @@ class streamsTest extends KyoTest:
         "none" in {
             assert(
                 Streams.initSeq(Seq(1, 2, 3)).collect {
-                    case v if false => ???
+                    case v if false => (): Unit < Streams[Int]
```

Comment:
Do we need a generalized unit method to improve inference?

```
object Kyo:
  def unit[S]: Unit < S = ()
```

Reply:
I'd say this is an edge case for testing. I think this compilation error would happen most often when users make mistakes for example not composing effects correctly.